src/shared/components/Select/Select.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { forwardRef } from 'react'
+import InputBase, { InputBaseProps } from '../InputBase'
+import { SelectButton, SelectMenu, SelectOption, SelectWrapper, StyledLabelText } from './Select.style'
+import { useSelect, UseSelectStateChange } from 'downshift'
+import Icon from '../Icon'
+
+export type SelectedItem = {
+  value: string
+  name: string
+} | null
```

Comment:
I think it may be more clear from consumer perspective if this type isn't nullable by itself. The `onChange` handler could  specify `| null` instead. What do you think?